frontend/src/containers/EditDetails.tsx

```diff
@@ -110,7 +115,9 @@ function EditDetails() {
                     id="topicAreaId"
                     name="topicAreaId"
                     label={settings.topicAreaLabels.singular}
-                    hint={`Select an existing ${settings.topicAreaLabels.singular.toLowerCase()}`}
+                    hint={`${t(
+                      "SelectExistingLeading"
```

Comment:
`t("SelectExistingLeading")` points to "**_s_**elect an existing". The s is lower case, perhaps we can just change the entry in the translation.json file? If we need it lower case in other places, maybe just make another entry?

Reply:
I can change that right now, give me few minutes